package.json

```diff
@@ -113,6 +114,7 @@
       "remark -qf"
     ],
     "*.{gn,gni}": [
+      "npm run gn-check",
```

Comment:
This depends on header files as well so this should be put in a new block `*.{gn,gni,h,cc,mm}`

Reply:
i thought about this but i was concerned about the lag time - it's not awful but slow enough that i wanted it to take more minimal effect 🤔